### DISCUSSION TITLE
How can I get Chinese label in confusion matrix?

### DISCUSSION THREAD

**Discussion by ChengYao0209** [ultralytics/yolov5#8775] [Q&A]
Hello

I train my model and I want to get some data/pic in my ppt
but my confusion matrix doesn't show my Chinese labels 
However, my predict result can show Chinese labels and bbox correctl

Is  there any way can solve this question?may add some encoding utf-8 ?

**Comment by glenn-jocher** [comment]
@ChengYao0209 thanks for the question! We've added support for Chinese characters in most places, but the confusion matrix is generated by matplotlib (which has problems) whereas the image annotations are done using PIL (works well). 

I'm not sure if there are any options for us to fix this, but you could try to debug the confusion matrix plotting code here. If you find a solution please submit a PR to help other users.

https://github.com/ultralytics/yolov5/blob/e309a855860bc3f618c3541909c515a65ffc35b0/utils/metrics.py#L187-L216
